src/Mod/Part/App/AppPart.cpp

```diff
@@ -498,6 +498,7 @@ PyMOD_INIT_FUNC(Part)
     Part::GeometryBoolExtension   	::init();
     Part::GeometryDoubleExtension 	::init();
     Part::GeometryMigrationExtension	::init();
+    Part::GeometryMigrationPersistenceExtension	::init();

```

Comment:
This line breaks column alignment of the file. This should be addressed somehow - IMHO we should stop column align stuff like this as it makes problems with formatting. @wwmayer ?

Reply:
In order to not break the formatting the block should be put into a
```
// clang-format off
// clang-format on

```block

Reply:
Sorry, I'm totally confused here and don't understand either the problem or the expected goals.  I can blindly toss the clang-format in around [something that I can't even scope], but I can't even define what the right block is in this case.  We have a couple of names that are too long for the alignment, but the overall idea holds.

Reply:
It's no longer part of this PR, so the point is moot, right?